WireAPI/Sources/WireAPI/Network/APIService/APIService.swift

```diff
@@ -49,10 +49,8 @@ public protocol APIServiceProtocol {
 
 public final class APIService: APIServiceProtocol {
```

Comment:
so are these 3 classes within a new module of WireAPI (Network)?
that would make sense to separate from actual endpoints

Reply:
these 3 would start to replace WireTransport right?

Reply:
yes that's a good idea, we can create a `WireNetwork` target inside the package, but I do that in another PR. And yes, this is the basis for replacing `WireTransport`.